Failed publish of node auto-instrumentation

https://github.com/open-telemetry/opentelemetry-operator/runs/4228736205?check_suite_focus=true

```bash
Run docker/build-push-action@v2.7.0
Docker info
/usr/bin/docker buildx build --build-arg version=0.27.0 --label org.opencontainers.image.title=opentelemetry-operator --label org.opencontainers.image.description=Kubernetes Operator for OpenTelemetry Collector --label org.opencontainers.image.url=https://github.com/open-telemetry/opentelemetry-operator --label org.opencontainers.image.source=https://github.com/open-telemetry/opentelemetry-operator --label org.opencontainers.image.version=0.27.0 --label org.opencontainers.image.created=2021-11-16T18:11:15.845Z --label org.opencontainers.image.revision=b64c1abec128cd2c1540249813a68c7fe25980e0 --label org.opencontainers.image.licenses=Apache-2.0 --tag ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.27.0 --tag ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:latest --iidfile /tmp/docker-build-push-kZwu51/iidfile --metadata-file /tmp/docker-build-push-kZwu51/metadata-file --push autoinstrumentation/nodejs
#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 198B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 transferring context: 58B done
#2 DONE 0.0s

#4 [internal] load metadata for docker.io/library/node:16
#4 DONE 0.0s

#5 [auth] library/busybox:pull token for registry-1.docker.io
#5 DONE 0.0s

#3 [internal] load metadata for docker.io/library/busybox:latest
#3 DONE 0.4s

#9 [internal] load build context
#9 transferring context: 2.16kB done
#9 DONE 0.0s

#7 [build 1/4] FROM docker.io/library/node:16
#7 DONE 0.0s

#6 [stage-1 1/2] FROM docker.io/library/busybox@sha256:e7157b6d7ebbe2cce5eaa8cfe8aa4fa82d173999b9f90a9ec42e57323546c353
#6 resolve docker.io/library/busybox@sha256:e7157b6d7ebbe2cce5eaa8cfe8aa4fa82d173999b9f90a9ec42e57323546c353 done
#6 sha256:e7157b6d7ebbe2cce5eaa8cfe8aa4fa82d173999b9f90a9ec42e57323546c353 2.29kB / 2.29kB done
#6 sha256:34efe68cca33507682b1673c851700ec66839ecf94d19b928176e20d20e02413 527B / 527B done
#6 sha256:7138284460ffa3bb6ee087344f5b051468b3f8697e2d1427bac1a20c8d168b14 1.46kB / 1.46kB done
#6 sha256:e685c5c858e36338a47c627763b50dfe6035b547f1f75f0d39753db71e319016 772.79kB / 772.79kB 0.1s
#6 sha256:e685c5c858e36338a47c627763b50dfe6035b547f1f75f0d39753db71e319016 772.79kB / 772.79kB 0.1s done
#6 extracting sha256:e685c5c858e36338a47c627763b50dfe6035b547f1f75f0d39753db71e319016
#6 ...

#8 [build 2/4] WORKDIR /operator-build
#8 DONE 0.3s

#10 [build 3/4] COPY . .
#10 DONE 0.0s

#6 [stage-1 1/2] FROM docker.io/library/busybox@sha256:e7157b6d7ebbe2cce5eaa8cfe8aa4fa82d173999b9f90a9ec42e57323546c353
#6 extracting sha256:e685c5c858e36338a47c627763b50dfe6035b547f1f75f0d39753db71e319016 0.2s done
#6 DONE 0.4s

#11 [build 4/4] RUN npm install
#11 13.08 npm notice 
#11 13.08 npm notice New patch version of npm available! 8.1.0 -> 8.1.3
#11 13.08 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.1.3>
#11 13.08 npm notice Run `npm install -g npm@8.1.3` to update!
#11 13.08 npm notice 
#11 13.09 npm ERR! code ETARGET
#11 13.09 npm ERR! notarget No matching version found for @opentelemetry/auto-instrumentations-node@0.27.0.
#11 13.09 npm ERR! notarget In most cases you or one of your dependencies are requesting
#11 13.09 npm ERR! notarget a package version that doesn't exist.
#11 13.10 
#11 13.10 npm ERR! A complete log of this run can be found in:
#11 13.10 npm ERR!     /root/.npm/_logs/2021-11-16T18_11_31_023Z-debug.log
#11 ERROR: executor failed running [/bin/sh -c npm install]: exit code: 1
------
 > [build 4/4] RUN npm install:
#11 13.08 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.1.3>
#11 13.08 npm notice Run `npm install -g npm@8.1.3` to update!
#11 13.08 npm notice 
#11 13.09 npm ERR! code ETARGET
#11 13.09 npm ERR! notarget No matching version found for @opentelemetry/auto-instrumentations-node@0.27.0.
#11 13.09 npm ERR! notarget In most cases you or one of your dependencies are requesting
#11 13.09 npm ERR! notarget a package version that doesn't exist.
#11 13.10 
#11 13.10 npm ERR! A complete log of this run can be found in:
#11 13.10 npm ERR!     /root/.npm/_logs/2021-11-16T18_11_31_023Z-debug.log
------
error: failed to solve: executor failed running [/bin/sh -c npm install]: exit code: 1
Error: buildx failed with: error: failed to solve: executor failed running [/bin/sh -c npm install]: exit code: 1
```

--- a/autoinstrumentation/nodejs/package.json
+++ b/autoinstrumentation/nodejs/package.json
@@ -15,8 +15,8 @@
   },
   "dependencies": {
     "@opentelemetry/api": "1.0.3",
-    "@opentelemetry/auto-instrumentations-node": "0.27.0",
-    "@opentelemetry/exporter-otlp-grpc": "0.27.0",
-    "@opentelemetry/sdk-node": "0.27.0"
+    "@opentelemetry/auto-instrumentations-node": "0.26.0",
+    "@opentelemetry/exporter-otlp-grpc": "0.26.0",
+    "@opentelemetry/sdk-node": "0.26.0"
   }
 }

--- a/versions.txt
+++ b/versions.txt
@@ -16,7 +16,7 @@ autoinstrumentation-java=1.7.0
 
 # Represents the current release of NodeJS instrumentation.
 # Should match value in autoinstrumentation/nodejs/package.json
-autoinstrumentation-nodejs=0.27.0
+autoinstrumentation-nodejs=0.26.0
 
 # Represents the current release of Python instrumentation.
 # Should match value in autoinstrumentation/python/requirements.txt

